languages.json

```
@@ -237,6 +237,13 @@
         "url": "https://github.com/PyGamer0/flax",
         "sbcs": true
     },
+    "Flip": {
```

The ID here needs to match the runner filename, and it's case-sensitive.

```suggestion
    "flip": {
```

---

Yeah, I never knew what the identifier in languages.json was actually for. Maybe you should clarify the `identifier-safe` thing in the contributing docs?

---

Ah, sorry, you're right, that's never specified anywhere.